README.rst

```diff
@@ -50,6 +50,10 @@ Basic usage:
     with etcd.lock('doot-machine') as lock:
         # do something
 
+    # locks with prefix
+    etcd.lock_prefix = "/locks"
+    lock = etcd.lock('thing')   # Lock will be generated under /locks/thing
```

Comment:
I'm not sure about this API. I'd rather `etcd.lock_prefix` wasn't directly used as a properly, instead you use `etcd.lock('thing', prefix='/locks')`, or `etcd3.client(lock_prefix='/locks')` only

Reply:
Makes sense. Changing it accordingly.